capstone/src/main/java/com/google/step/servlets/PageViewServlet.java

```diff
@@ -54,7 +54,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         String json;
         if (restaurantKey == null && year <= 0) {
-            List<RestaurantPageViews> restaurantPageViewsList = metricsClient.getAllPageViews();
+            List<RestaurantPageViews> restaurantPageViewsList = metricsClient.getAllPageViews("restaurantName");
```

Comment:
hmmm, to help decouple things, I do think this sort of thing really should be an enum. Then, you don't have to require callers of your client to know the property name on an entity.

Reply:
Same comment as above.